src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
```

Comment:
Just destructure `person`. If you don't like using like `person.name or person.age`, then you can do following
```
export const Person = ({ person }) => {
  const { name, age } = person;
  
  return (
    // Your JSX code
  )
}

Reply:
👍